molecule/driver/docker.py

```diff
@@ -87,6 +88,26 @@ class Docker(base.Base):
             buildargs:
                 http_proxy: http://proxy.example.com:8080/
 
+    When attempting to utilize a container image with `systemd`_ as your init
+    system inside the container to simulate a real machine, make sure to set
+    the ``privileged``, ``volume_mounts``, ``command``, and ``environment``
+    values. An example using the ``centos:7`` image is below:
+
+    .. note:: Do note that running containers in privileged mode is considerably
+              less secure. For details, please reference `Docker Security
+              Configuration`_
+
+    .. code-block:: yaml
+
+        platforms:
+        - name: instance
+            image: centos:7
+            privileged: true
+            volume_mounts:
+            - "/sys/fs/cgroup:/sys/fs/cgroup:rw"
+            command: "/usr/sbin/init"
+            environment: { container: docker }
```

Comment:
I don't like it inline, could you change it to multi-line plz?

Reply:
@webknjaz fixed :)